lib/fortnox/api/models/base.rb

```diff
@@ -87,17 +76,9 @@ def parent
           @parent || self.class.new(self.class::STUB.dup)
         end
 
-        def to_hash(recursive = false)
-          return super() if recursive
-
-          self.class.schema.keys.each_with_object({}) do |key, result|
-            result[key] = self[key]
-          end
-        end
-
         # dry-types filter anything that isn't specified as an attribute on the
         # class that is being instantiated. This wrapper preserves the meta
-        # properties we need to track object state during that initilisation and
+        # properties we need to track object state during that initialization and
```

Comment:
Should be `initialisation`, with an `s`, since we/I standardise on British english...